extension/src/plots/index.ts

```diff
@@ -223,6 +219,19 @@ export class Plots extends BaseRepository<TPlotsData> {
     this.deferred.resolve()
   }
 
+  private triggerDataUpdate() {
+    void this.data.managedUpdate()
+  }
+
+  private onDidTriggerDataUpdate() {
+    const sendCachedDataToWebview = () => {
+      this.plots.resetFetched()
+      this.plots.setComparisonOrder()
```

Comment:
[F] Resetting the fetchedRevs means that we get spinners in the ribbon for all of the revisions. Setting the comparison order means we get a loading state for the comparison table:

https://user-images.githubusercontent.com/37993418/227394044-b661259a-4440-4d5a-b952-bbe74e1c9d71.mov